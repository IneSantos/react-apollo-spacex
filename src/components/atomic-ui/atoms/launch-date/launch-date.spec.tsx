import React from "react";
import { render } from "@testing-library/react";
import LaunchDate from "./launch-date";

describe("LaunchDate atom tests", () => {
  test(`LaunchDate component render`, () => {
    const { container } = render(<LaunchDate date="2022/04/10" />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
