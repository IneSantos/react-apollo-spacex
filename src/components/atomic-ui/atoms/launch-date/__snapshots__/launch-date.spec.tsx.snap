// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LaunchDate atom tests LaunchDate component render 1`] = `
<div
  class="launch-date"
>
  April 10, 2022
</div>
`;
